### DISCUSSION TITLE
Coinbase Wallet Connector doesn't work

### DISCUSSION THREAD

**Discussion by achille1017** [wevm/wagmi#2913] [Q&A]
async function connectCB() {
    const result = await connect({
        connector: new CoinbaseWalletConnector({
            options: {
                chains,
                appName: 'wallet-link',
                //jsonRpcUrl: 'https://rpc.ankr.com/eth',
            },
        })
    })
    console.log(result)
    //await signMessageB()
}
const btnCB = document.querySelector('#connectCB');
btnCB.addEventListener("click", connectCB);

When I click on #connectCB, the console tells me
Uncaught (in promise) ReferenceError: process is not defined
    hR CoinbaseWalletSDK.js:13
    x out.js:1
    rce index.js:6
    x out.js:1
    t coinbaseWallet.js:106
    promise callback*getProvider coinbaseWallet.js:106
    connect coinbaseWallet.js:56
    Od chunk-LAFZBYO7.js:657
    PZe main.js:68
    EventListener.handleEvent* main.js:81
    <anonymous> main.js:167

**Comment by glitch-txs** [comment]
this is an issue on Coinbase SDK, you can do some workaround by adding a process polyfill on tope of your main js file
